Algorithms/Python/Sorting/bubble_sort.py

```diff
@@ -0,0 +1,38 @@
+"""
+    Bubble sort is a sorting algorithm in which every element is compared and swapped so that the last index is filled
+    with the correct element first
+"""
+
+
+def bubble_sort(arr, order=None):
+    """
+    :param arr: array which needs to be sorted
+    :param order: Order of sorting, '1' for Descending, otherwise Ascending
+    :return: Sorted Array is returned
+
+    Time Complexity:  O(n^2)
+    Space Complexity: O(1)
+    """
+    arr_len = len(arr)
+    for i in range(arr_len - 1):
+        for j in range(arr_len - 1 - i):
+            if order == '1':
+                if arr[j] < arr[j + 1]:
+                    arr[j + 1], arr[j] = arr[j], arr[j + 1]
+            else:
+                if arr[j] > arr[j + 1]:
+                    arr[j + 1], arr[j] = arr[j], arr[j + 1]
+    return arr
+
+
+if __name__ == "__main__":
+    n = input("Enter space separated values of the array: ")
+    input_arr = list()
+    for ele in n.split(" "):
```

Comment:
Add input validation to ensure that the list contains comparable elements before sorting.